model/service.go

```diff
@@ -54,6 +54,9 @@ type Service struct {
 	// service DNS name.  External services are name-based solution to represent
 	// external service instances as a service inside the cluster.
 	ExternalName string `json:"external"`
+
+	// Annotations on the service.
+	Annotations map[string]string `json:"annotations,omitempty"`
```

Comment:
Good point. I don't know if annotation makes sense outside of K8s. I can remove this from model and keep all the logic in platform/kube.